### ISSUE TITLE
Create OAI-PMH set for Murray dataverse

### ISSUE THREAD

**Issue by sbarbosadataverse** [IQSS/dataverse.harvard.edu#68]
need a set for harvesting  by HOLLIS, Harvard Library

**Comment by jggautier** [comment]
The set is created now for datasets in the Murray dataverse:

<img width="956" alt="Screen Shot 2020-04-06 at 5 18 09 PM" src="https://user-images.githubusercontent.com/18374574/78606059-ac209f00-782a-11ea-8f95-cc888e96f4d2.png">

It looks like Dataverse updates sets automatically each day. When new datasets are published, the set should be added by the next day.

Ceilyn Boyd emailed Robin Wendler from Harvard Library about setting up harvesting, who wrote that someone else would need to help, and also shared that:

> The harvest needs to be configured and the metadata mapped to Primo fields, so it won’t happen automatically.  It’s recommended that all work requests be submitted through Footprints, with as much information as possible. In this case, that would probably include the OAI data provider endpoint, the setName and setSpec, metadata profile [fields and usage], metadata serialization options [simple DC only or richer formats like DDI]).

I'm working to figure out how we can help whoever at Harvard Library could help set this up (mapping the fields, configuring harvesting).

Also including needed information here for reference:

1. OAI data provider endpoint: https://dataverse.harvard.edu/oai
2. setSpec: Murray_Research_Archive
3. setName: Murray_Research_Archive
4. metadata profile: the [Dataverse crosswalk](https://docs.google.com/spreadsheets/d/10Luzti7svVTVKTA-px27oq3RxCUM-QbiTkm8iMd5C54/edit#gid=0) contains the fields, field definitions and how they map to other standards. The [metadatablock definitions](https://docs.google.com/spreadsheets/d/13HP-jI_cwLDHBetn9UKTREPJ_F4iHdAvhjmlvmYdSSw) contain additional information about the fields (e.g. which fields are required to create a dataset, what the allowed values types for each field are, which fields have parent-child relationships with each other)
5. metadata serialization options: https://dataverse.harvard.edu/oai?verb=ListMetadataFormats

**Comment by jggautier** [comment]
Just an update: Early in April month Ceilyn connected with others from Harvard Library to request that the Murray_Research_Archive set be harvested into Hollis. We're waiting to hear back.

**Comment by jggautier** [comment]
Another update: In April 2021 Corinna Baksik, Senior Systems Librarian at Harvard Library, let us know that their library discovery system Primo was able to use the harvesting set to retrieve all of the Dataverse collection's records, and that they'll  "index them using a default Dublin Core template and see what needs to be modified for this set."

**Comment by jggautier** [comment]
"Original" Murray datasets were moved into a new collection so that it would be easy to create a harvesting set that contains only the original Murray datasets, but when I try to create that harvesting set, the Harvard Dataverse Repository tells me that the search query returned no results:

<img width="1007" alt="Screen Shot 2021-09-27 at 4 04 47 PM" src="https://user-images.githubusercontent.com/18374574/134977353-07b8239d-12e3-46d2-999b-5f7f7d206626.png">

5112855 is the database ID of the Dataverse collection at https://dataverse.harvard.edu/dataverse/originalMRA, create in early Sept. 2021, that contains the "original" Murray datasets.

@sbarbosadataverse I'm going to open another issue to report this and describe some of the troubleshooting I tried.
